Sources/App/Commands/Analyze.swift

```diff
@@ -724,7 +725,10 @@ extension Analyze {
             logger.warning("Twitter.postToFirehose failed: \(error.localizedDescription)")
         }
     }
+    
 
+    
+    
```

Comment:
nit: formatting (excessive whitespace)

@daveverwer: We don't have a format/linter specified, right? Are we against adding one to make it easier for folks to contribute while maintaining a common style? A small bonus is that [SwiftFormat](https://github.com/nicklockwood/SwiftFormat) will handle all the file headers automatically which I know you said you were going to update.